spec.html

```diff
@@ -28312,238 +28312,246 @@
+                  <td>
+                    `L`
+                  </td>
+                  <td>
+                    `M`
+                  </td>
+                  <td>
+                    `N`
+                  </td>
+                  <td>
+                    `O`
+                  </td>
+                  <td>
+                    `P`
+                  </td>
+                  <td>
+                    `Q`
+                  </td>
+                  <td>
+                    `R`
+                  </td>
+                  <td>
+                    `S`
+                  </td>
+                  <td>
+                    `T`
+                  </td>
+                  <td>
+                    `U`
+                  </td>
+                  <td>
+                    `V`
+                  </td>
+                  <td>
+                    `W`
+                  </td>
+                  <td>
+                    `X`
+                  </td>
+                  <td>
+                    `Y`
+                  </td>
+                  <td>
+                    `Z`
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    `0`
+                  </td>
+                  <td>
+                    `1`
+                  </td>
+                  <td>
+                    `2`
+                  </td>
+                  <td>
+                    `3`
+                  </td>
+                  <td>
+                    `4`
+                  </td>
+                  <td>
+                    `5`
+                  </td>
+                  <td>
+                    `6`
+                  </td>
+                  <td>
+                    `7`
+                  </td>
+                  <td>
+                    `8`
+                  </td>
+                  <td>
+                    `9`
+                  </td>
+                  <td>
+                    `_`
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </figure>
+          1. If  _Unicode_ is *true*, then
+            1. Create an empty set _U_.
+            1. For every character _c_ not in set _A_ where Canonicalize(_c_) is in _A_, add _c_ to _U_.
```

Comment:
The change I'm proposing specifically doesn't take into account _ignoreCase_ as we want the set to be the same with and without _ignoreCase_.